features/path_rules.feature

```diff
@@ -0,0 +1,84 @@
+@sig-network @conformance @release-1.19
+Feature: Path rules
+
+  An Ingress may define routing rules based on the request path.
+
+  If the HTTP request path matches one of the paths in the
+  Ingress objects, the traffic is routed to its backend service.
+
+  Background:
+    Given a new random namespace
+    Given an Ingress resource
+      """
+      apiVersion: networking.k8s.io/v1
+      kind: Ingress
+      metadata:
+        name: path-rules
+      spec:
+        rules:
+          - host: "path-rules"
```

Comment:
Do we want to put an giant block or separate these into different cases?
The invalid path will cause the entire Ingress object to be rejected by most controllers...

Reply:
Agree that some way to test small sets of rules without all the boilerplate could be helpful.

Reply:
We can reduce the boilerplate for simples cases with a limited set of rules.

I can also imagine more complex setups where we'd want to test multiple path rules to validate matches are done according to the spec: `Exact` paths are preferred over `Prefix`.